workspaces/src/network/mainnet.rs

```diff
@@ -3,10 +3,11 @@ use std::path::PathBuf;
 use async_trait::async_trait;
 
 use crate::network::Info;
-use crate::network::{Account, CallExecution, NetworkClient, NetworkInfo, TopLevelAccountCreator};
+use crate::network::{NetworkClient, NetworkInfo, TopLevelAccountCreator};
+use crate::result::CallExecution;
```

Comment:
Why is this type not exposed?  It is returned by exposed functions.  So if a user wants to make a function that returns the result of one of the exposed function they reference the type.

Reply:
What are you referring to? This is just a type that wraps the type returned that includes metadata

Reply:
Sorry I meant from the crate: https://github.com/near/workspaces-rs/pull/102/files#diff-f2509a15c16bf20cbd905fc2d19c6b77363687408bf1020327da68936ec19ef9

Account exposes `CallExecutionDetails` and `CallExecution`, but they aren't exported.

Reply:
those are exposed under the `result` module which we have exported via `pub mod result;` in `lib.rs`

Reply:
Ah I see thanks!